src/shared/lib/markDownRules.tsx

```diff
@@ -418,12 +431,28 @@ const markDownRules: RenderRules = {
 
     const isCached = !!CacheManager.entries[node.attributes.src];
 
+    if (!imageSize.width || !imageSize.height) {
+      const imageDimensions = getImageSize(src);
+
+      if (!imageDimensions) {
+        throw Error(
+          `[markDownRules.image]: No image dimension found for url: ${src}`,
+        );
+      }
```

Comment:
Handling this error should be part of `getImageSize`, also should this instead fail gracefully and let MarkdownView render the image as is?